packages/material-ui-lab/src/Pagination/Pagination.d.ts

```diff
@@ -1,2 +1,100 @@
-declare const Pagination: any;
-export default Pagination;
+import * as React from 'react';
+import { StandardProps } from '@material-ui/core';
+import { UsePaginationItem } from './usePagination';
+
+export interface PaginationProps
+  extends StandardProps<
+    React.HTMLAttributes<HTMLElement>,
+    PaginationClassKey,
+    'onChange' | 'children'
+  > {
+  /**
+   * Number of always visible pages at the beginning and end.
+   */
+  boundaryCount?: number;
+  /**
+   * The active color.
+   */
+  color?: 'default' | 'primary' | 'secondary';
+  /**
+   * The total number of pages.
+   */
+  count?: number;
+  /**
+   * The page selected by default when the component is uncontrolled.
+   */
+  defaultPage?: number;
+  /**
+   * If `true`, the pagination will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * Accepts a function which returns a string value that provides a user-friendly name for the current page.
+   *
+   * For localization purposes, you can use the provided [translations](/guides/localization/).
+   *
+   * @param {string} [type = page] The link or button type to format ('page' | 'first' | 'last' | 'next' | 'previous').
```

Comment:
Could you remove the types in the jsdoc? They duplicate the TS types and conflict in this case. It's also not clear when `type` can be `undefined` while also having a default value.

Reply:
Ah, I thought that syntax somehow implied an optional argument. Will fix it in a sec.

Reply:
I can't remove the types from the jsdoc here. These docs are copied to the JS component by `yarn proptypes --disable-cache`, and then the `yarn docs:api` command will fail because there is no type annotation.